### ISSUE TITLE
Comms Iteration w/c 20 March 2023

### ISSUE THREAD

**Issue by rufuspollock** [life-itself/community#365]
# Iteration Goals

- [ ] Projects pages #97 
  - [x] All individual  projects pages **✅2023-03-29 all done up to priority 4**
  - [x] index page 🚧 **Page is created with simple list of projects, but not showing up on the website currently. Needs symlinking**
- [ ] New landing page #335 🚧 **#335 for basic wireframe with text** **💬2023-03-30 have made very good progress in https://lifeitself.org/home-new. 80% done.**
- [x] #344 ✅ 
  - [x] Community meeting on 23rd **✅2023-03-23 this took place. good recording from Valerie.**
- [x] #386 🚧 **Newsletter almost ready, waiting on praxis for final information** **✅2023-03-30 - sent but issue in send dout**
- [ ] #124 

---

## Capacity for Iteration

- Nathen: 24 Hours
- Zaib: 25 Hours

---

# 2023-03-16 - Iteration planning

Present: Nathen, Rufus, Zaib

Intention: co-work on projects and landing page plus plan backlog

- [x] Have a plan for next 2w iteration
  - [x] capacity ✅ **Included about and in metadata**
  - [ ] assigned items
  - [ ] 🏆 groomed backlog of more issues
- [ ] Projects pages
  - [x] do 3 projects pages
  - [ ] 🏆 index page
- [ ] Landing page
- [ ] ? Nathen can do some styling of stuff (not dependent on khalil)

🏆 = bonus / extra / stretch goal

Agenda

- [x] 🧘 Meditation
- [x] 👋 Check-in
- [x] Create agenda
- [x] Review agenda
  - [x] Recap from last time (see below)
  - [ ] Projects work https://github.com/life-itself/community/issues/97
  - [ ] Backlog prep 
  - [ ] Landing page https://github.com/life-itself/community/issues/335
  - [ ] Key Ideas page https://github.com/life-itself/community/issues/124
  - [ ] 🏆 a project for 2nd half of year e.g. conscious communities, conscious parenting (2nd half of year?)
  - [ ] AOB
    - [ ] Why markdown and hackmd
    - [x] Rufus and Nathen short 1 on 1 catchup
    - [x] 🏆 Curious about Conscious Parenting Direction/Timeline (if we have time) **⏭️ plan is next is to create a course**
- [x] Check-out and next actions
  - [ ] Zaib to post all the projects with score 5 and above

## Actions

AOB

- [ ] Obsidian guide
  - [ ] Set up templates folder
  - [ ] Set up assets folder
  - [ ] Tabs setup: spaces at 2
- [x] Log this meeting video with some tags (update calendar title e.g. `🎥 #tutorial #obsidian projects setup`

### Projects work

- [x] Update project template (see updated version below)
- [x] Zaib to update current project pages with new metadata

**Comment by rufuspollock** [comment]
@ZybN @nathenf FYI i copied meeting notes here and started pulling out goals for next iteration (starting next monday).

**Comment by Liyubov** [comment]
related to the previous question from rufus.
dear @nathenf, I wanted to ask about the edits of the interview: we have one audio interview with Life itself Praxis hub from the current residency organizer, Boaz. I was wondering if you would be able to help with it, since it could really contribute to the community and be really important contribution.
thank you!

**Comment by rufuspollock** [comment]
@Liyubov are you making a request for help from comms on that? (I'm assuming so). If so, a great first step would be to open an issue (or a discussion in fact) where you summarize what you want: suggest using the 5Ws: what, why, where, when, who (and how) e.g. want to publish an interview with boaz, because XXX, where: on blog / youtube, when: by XX date, who: Boaz is interviewee and you want Y person will do Z, ...

**Comment by rufuspollock** [comment]
FIXED. We made a lot of good progress. Achieved 80% + of goals.
